<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUI_StreamRecorder</class>
 <widget class="QWidget" name="GUI_StreamRecorder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>188</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Stream Recorder</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="9" column="0" colspan="4">
    <widget class="QCheckBox" name="cb_complete_tracks">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Only complete tracks</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="4">
    <widget class="QCheckBox" name="cb_create_playlist">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Create playlist</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="4">
    <widget class="QCheckBox" name="cb_activate">
     <property name="text">
      <string>Activate</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Target path:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lab_icon">
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Stream Recorder</string>
     </property>
    </widget>
   </item>
   <item row="12" column="3">
    <widget class="QPushButton" name="btn_ok">
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lab_path">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_path">
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cb_activate</sender>
   <signal>toggled(bool)</signal>
   <receiver>cb_complete_tracks</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cb_activate</sender>
   <signal>toggled(bool)</signal>
   <receiver>btn_path</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>78</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
